Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Update references to btrfs-tools instead of btrfs-progs (upstream name) in
 manpages (Closes: #599111).

--- a/man/btrfs.8.in
+++ b/man/btrfs.8.in
@@ -752,7 +752,7 @@ case of failure.
 
 .SH AVAILABILITY
 .B btrfs
-is part of btrfs-progs. Btrfs filesystem is currently under heavy development,
+is part of btrfs-tools. Btrfs filesystem is currently under heavy development,
 and not suitable for any uses other than benchmarking and review.
 Please refer to the btrfs wiki http://btrfs.wiki.kernel.org for
 further details.
--- a/man/btrfs-image.8.in
+++ b/man/btrfs-image.8.in
@@ -44,7 +44,7 @@ option if your extent tree is corrupted to make sure that all of the metadata is
 captured.
 .SH AVAILABILITY
 .B btrfs-image
-is part of btrfs-progs. Btrfs is currently under heavy development,
+is part of btrfs-tools. Btrfs is currently under heavy development,
 and not suitable for any uses other than benchmarking and review.
 Please refer to the btrfs wiki
 http://btrfs.wiki.kernel.org for further details.
--- a/man/mkfs.btrfs.8.in
+++ b/man/mkfs.btrfs.8.in
@@ -97,7 +97,7 @@ As default the unit is the byte, however it is possible to append a suffix
 to the arguments like \fBk\fP for KBytes, \fBm\fP for MBytes...
 .SH AVAILABILITY
 .B mkfs.btrfs
-is part of btrfs-progs. Btrfs is currently under heavy development,
+is part of btrfs-tools. Btrfs is currently under heavy development,
 and not suitable for any uses other than benchmarking and review.
 Please refer to the btrfs wiki
 http://btrfs.wiki.kernel.org for further details.
